hazelcast-documentation/src/RingBuffer.md

```diff
@@ -1,47 +1,233 @@
 ## RingBuffer
 
+The RingBuffer is a data structure where the data is stored in a ring-like structure. You can think of it as a circular array with a
+certain capacity. Each element in a RingBuffer can be accessed using a sequence ID. This ID is between the head and tail (inclusive) 
+of the RingBuffer. Each ringbuffer has a tail and a head. The tail is where the items are added and the head is where items get over
+written or get expired.
 
```

Comment:
You might want to add a picture how a ringbuffer works, the current explanation may not work for people with no idea what a RB is at all.